### PR TITLE
Create a new network for each instance of TestingKafka

### PR DESCRIPTION
**Pull request by Praveen2112**
Fixes https://github.com/trinodb/trino/issues/6412

### CODE REVIEW

**Comment by martint** [comment]
What’s the motivation? Please add some details to the commit message and PR

**Review by ssheikin — APPROVED** [review]

**Comment by Praveen2112** [comment]
@martint It is more on fixing the flaky tests related to Kafka.

File: testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java [hunk 1]
```diff
@@ -83,11 +83,12 @@ public static TestingKafka createWithSchemaRegistry()
     private TestingKafka(String confluentPlatformVersion, boolean withSchemaRegistry)
     {
         this.withSchemaRegistry = withSchemaRegistry;
+        Network network = Network.newNetwork();
```
Comment (kokosing) [inline testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java:86]: Do you need to close this?

**Review by kokosing — APPROVED** [review]

File: testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java (hunk 1 above)
Comment (Praveen2112) [reply testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java:86]: yes.

**Pull request merged**